README.md

```diff
@@ -110,10 +110,15 @@ You can also list all available commands with:
 $> ./myEnvDynawo.sh help
 ```
 
-We advise you to create an alias in your bashrc (or other) to be able to call Dyna&omega;o from anywhere:
+**We advise you to deploy our autocompletion** script to help you with the available commands, it will also set an alias in your bashrc or zshrc to be able to call Dyna&omega;o from anywhere. You can launch one of the two following commands:
```

Comment:
Please specify the command used to launch dynawo ("dynawo")

Reply:
ok